plugins/quetz_content_trust/quetz_content_trust/repo_signer.py

```diff
@@ -1,15 +1,12 @@
 import os
-import shutil
 from pathlib import Path
 
 import conda_content_trust.signing as cct_signing
 
 
 class RepoSigner:
     def sign_repodata(self, repodata_fn, pkg_mgr_key):
-        final_fn = self.in_folder / "repodata_signed.json"
-        print("copy", repodata_fn, final_fn)
-        shutil.copyfile(repodata_fn, final_fn)
+        final_fn = self.in_folder / "repodata.json"
 
         cct_signing.sign_all_in_repodata(str(final_fn), pkg_mgr_key)
```

Comment:
We probably need a variant to that function to produce signatures for all keys in one go.
It would be nice to also open a PR with this extra function to `conda-content-trust` repo.